apps/meteor/app/lib/server/functions/deleteMessage.ts

```diff
@@ -43,6 +43,7 @@ export async function deleteMessage(message: IMessage, user: IUser): Promise<voi
 
 	if (deletedMsg?.tmid) {
 		await Messages.decreaseReplyCountById(deletedMsg.tmid, -1);
+		await ReadReceipts.incrementThreadMessagesCountById(deletedMsg.tmid, -1);
```

Comment:
Kinda confusing the name is `increment` but is actually `decrementing` 👀

Reply:
Done, thankss!